include/aws/cryptosdk/session.h

```diff
@@ -56,6 +56,22 @@ struct aws_cryptosdk_session;
 
 enum aws_cryptosdk_mode { AWS_CRYPTOSDK_ENCRYPT = 0x9000, AWS_CRYPTOSDK_DECRYPT = 0x9001 };
 
+/**
+ * Creates a new encryption or decryption session.
+ *
+ * @return The new session, or NULL on failure (in which case, an AWS error code is set)
```

Comment:
"The new session object.."

Reply:
Calling it a session object is not consistent with the terminology we have been using throughout the API documentation. Note that this is not new wording in this PR. I copied the exact wording already used in the other session constructor and just modified the description of the single changed parameter.

Reply:
The reason I suggested `session object` is that the in trailing lines we use that terminology.